webapp/components/user_settings/user_settings_security.jsx

```diff
@@ -74,6 +55,16 @@ class SecurityTab extends React.Component {
         };
     }
 
+    componentWillMount() {
```

Comment:
We shouldn't use `componentWillMount` since it's basically the constructor and React recommends performing this type of initial request in `componentDidMount` (you'll have to disable the eslint rule for those two set state lines, but that's OK because it's in the callbacks).

See here for an example: https://facebook.github.io/react/tips/initial-ajax.html